images/linux/scripts/installers/postgresql.sh

```diff
@@ -1,18 +1,22 @@
 #!/bin/bash -e
 ################################################################################
 ##  File:  postgresql.sh
-##  Desc:  Installs Postgresql
+##  Desc:  Installs PostgreSQL
 ################################################################################
 
 REPO_URL="https://apt.postgresql.org/pub/repos/apt/"
 
-#Preparing repo for PostgreSQL 12.
+# Preparing repo for PostgreSQL
 wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
-echo "deb $REPO_URL `lsb_release -cs`-pgdg main" | tee /etc/apt/sources.list.d/pgdg.list
+echo "deb $REPO_URL $(getOSVersionLabel)-pgdg main" | tee /etc/apt/sources.list.d/pgdg.list
```

Comment:
You should import this function before using from a helper script.

Reply:
Thanks. Done